tests/integration/dataframe/test_dataframe.py

```diff
@@ -615,3 +628,217 @@ def test__export_table_data__works(self, client: DataFrameClient, create_table):
                 response.read()
                 == b'"col1","col2","col3"\r\n1,2.5,6.5\r\n2,1.5,5.5\r\n3,2.5,7.5'
             )
+
+    def test__append_table_data__append_request_success(
```

Comment:
Can we have these tests read back the table metadata and verify the `RowCount` and `SupportsAppend` are correct?

Reply:
I intended this comment to apply to all of the tests that are only calling `append_table_data` without any asserts, not just `test__append_table_data__flush_only_with_none`.